enginetest/queries/json_table_queries.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2022 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package queries
+
+import (
+	"github.com/dolthub/go-mysql-server/sql"
+)
+
+var JSONTableQueryTests = []QueryTest{
```

Comment:
Need some tests joining json tables to physical tables

If they don't work, add skipped tests

Reply:
Also tests where the expression for json data comes from a column, not a string literal